litellm/model_prices_and_context_window_backup.json

```diff
@@ -2168,6 +2168,28 @@
         "supports_vision": true,
         "source": "https://cloud.google.com/vertex-ai/generative-ai/docs/learn/models#foundation_models"
     },
+    "gemini/gemini-gemma-2-27b-it": {
```

Comment:
@ishaan-jaff where do you see these? 

I just tried to make a query to google ai studio with this model name and it failed - 
<img width="937" alt="Screenshot 2024-11-26 at 4 34 59 PM" src="https://github.com/user-attachments/assets/4fe923ad-87e3-424a-8269-3af4b076c80f">